charts/filsnap/templates/_helpers.tpl

```diff
@@ -0,0 +1,72 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "filsnap.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- define "filsnap._priv.logLevelNamed" -}}
+{{- range .Values.nodelocker.logging.named }}{{(print .logger ":" .level ) }},{{- end }}
+{{- end }}
+
+{{- define "filsnap.logLevelNamed" -}}
+{{- if .Values.nodelocker.logging.named }}
+{{- (include "filsnap._priv.logLevelNamed" .) | trimSuffix "," }}
```

Comment:
The `filsnap._priv.logLevelNamed` does the map / join of this array of objects. It needs to be a seperate function I believe, as I'm not sure how to include the `| trimSuffix ","` along side the `range` template stuff.